libexec/python/utils.py

```diff
@@ -172,19 +172,22 @@ def api_get(url,data=None,default_header=True,headers=None,stream=None,return_re
 
     return stream
 
+
 def basic_auth_header(username, password):
     '''basic_auth_header will return a base64 encoded header object to
     generate a token
     :param username: the username
     :param password: the password
     '''
     s = "%s:%s" % (username, password)
+    logger.debug("AUTH: Input username/password: %s", s)
```

Comment:
oh, ok. I mentioned adding these in our discussion a few times, and you didn't say anything, so I thought that was the plan moving forward. I'm definitely glad to remove all prints and debug another way.